### DISCUSSION TITLE
Cannot query docs of supabase graphql api

### DISCUSSION THREAD

**Discussion by Tjerk-Haaye-Henricus** [supabase/supabase#15641] [Questions]
Hey there, 

did anyone faced this issue already. It's on a completely new created project.

<img width="551" alt="image" src="https://github.com/supabase/supabase/assets/44015586/4b47ac26-ea93-404b-9137-f0f8a6b27068">

**Comment by GaryAustin1** [comment]
There is a new open issue on this... https://github.com/supabase/pg_graphql/issues/377
The conflict appears to be with Vault/pg_sodium.
